weed/filer/ydb/ydb_store.go

```diff
@@ -112,14 +110,13 @@ func (store *YdbStore) initialize(dirBuckets string, dsn string, tablePathPrefix
 func (store *YdbStore) doTxOrDB(ctx context.Context, query *string, params *table.QueryParameters, tc *table.TransactionControl, processResultFunc func(res result.Result) error) (err error) {
 	var res result.Result
 	if tx, ok := ctx.Value("tx").(table.Transaction); ok {
-		res, err = tx.Execute(ctx, *query, params, options.WithQueryCachePolicy(options.WithQueryCachePolicyKeepInCache()))
+		res, err = tx.Execute(ctx, *query, params)
```

Comment:
Why you remove query cache option?

Reply:
> Why you remove query cache option?

This option in New SDK version is depricated. Query cache enable by default.